hikari/events/shard_events.py

```diff
@@ -249,7 +249,7 @@ def __getitem__(self, index_or_slice: slice, /) -> typing.Sequence[guilds.Member
     def __getitem__(
         self, index_or_slice: typing.Union[int, slice], /
     ) -> typing.Union[guilds.Member, typing.Sequence[guilds.Member]]:
-        return collections.get_index_or_slice(self.members, index_or_slice)
+        return collection_types.get_index_or_slice(self.members, index_or_slice)
```

Comment:
i needed a name that wasn't collections, as I had some classpath issues occuring while testing this where I couldn't see the real issue since this was shadowing collections in the standard lib.

Turned out to be a pycharm issue that i fixed. I forgot to revert it.